src/components/biggive-text-input/biggive-text-input.stories.tsx

```diff
@@ -21,15 +21,18 @@ const Template = (args: Args) => `
     ${args.input}
 </biggive-text-input>`;
 
-export const TextInput = Template.bind({});
-TextInput.args = {
-  currency: 'GBP',
-  spaceBelow: 0,
-  selectStyle: 'bordered',
-  input: `<input
-    id="enter-donation-amount"
-    slot="input"
-    value="60.00"
-    />`,
-  label: `<label for="enter-donation-amount" slot="label">Enter your donation amount</label>`,
-} as Args;
+export const TextInput = {
+  render: Template,
+
+  args: {
+    currency: 'GBP',
+    spaceBelow: 0,
+    selectStyle: 'bordered',
+    input: `<input
+      id="enter-donation-amount"
+      slot="input"
+      value="60.00"
+      />`,
+    label: `<label for="enter-donation-amount" slot="label">Enter your donation amount</label>`,
+  } as Args,
```

Comment:
not sure what the original reason for `as Args` was but it doesn't look like we need it.
```suggestion
  },
```

I know this you're only making a whitespace change on this line, just happened to notice since it shows in the diff.

Reply:
@bdsl This is a bulk migration to the new Storybook format, so it was probably needed before but not any more. https://github.com/thebiggive/components/pull/593/commits/9639b4b229779e02940edf9ede47934a872577cd